e2e_testing/torchscript/reduction.py

```diff
@@ -122,3 +122,63 @@ def forward(self, a):
 @register_test_case(module_factory=lambda: ReduceMeanDtypeModule())
 def ReduceMeanDtypeModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4, 5).to(torch.float64))
+
+# ==============================================================================
+
+class ReduceMaxAlongDim(torch.nn.Module):
+
+  def __init__(self):
+    super().__init__()
+
+  @export
+  @annotate_args([
+      None,
+      ([-1, -1, -1], torch.float64, True),
+  ])
+  def forward(self, a):
+    return torch.max(a, 1)[0]
```

Comment:
Is there a reason we can't return the two tensors?

Reply:
We don't have the support for the combination of memref<f64> and memref<i64> in the refbackend.  We do have memref of f32 and i64 which I have added as a third test case.